Automattic/jetpack
# Admin: add new About page.

For now it's just a skeleton with the wrappers needed and without the nav buttons. Needs content and styles.

Details in p1HpG7-6v8-p2

<img width="1137" alt="Captura de pantalla 2019-03-29 a la(s) 11 27 45" src="https://user-images.githubusercontent.com/1041600/55239626-d2722e80-5215-11e9-9a57-907b1cee38c5.png">



#### Changes proposed in this Pull Request:


* Introduce new About page as a submenu of the **Jetpack** main menu item.
* Introduce new parameter `show-nav` for `Jetpack_Admin_Page::wrap_ui` so buttons can be hidden when passing `false`.
* Introduce new class `Jetpack_About_Page` to implement the **About** page

#### Testing instructions:





* Go to the Jetpack menu and click the About page. It should display the page properly formatted as the mockups
* Interactive elements should work
* Tracks should be recorded

#### Proposed changelog entry for your changes:


* Introduce new About page.
